gulpfile.js

```diff
@@ -215,10 +217,30 @@ const test = gulp.series(testUnitTests, testWeb);
 const testInt = gulp.series(testDebugger);
 
 async function packageVsix() {
-    fs.emptyDirSync(packagedir);
-    return vsce.createVSIX({
+    const stanardHeader = '# Power Platform Extension';
+    const previewHeader = '# Power Platform Tools [PREVIEW]\n\n## This extension is used for internal testing against targets such as vscode.dev which require Marketplace published extensions, and is not supported.';
```

Comment:
fine to have this text inline for now; but should we decide to add more disclaimer text, we'd need to factor the text out into a disclaimer.md